bin/create_pdf_template.py

```diff
@@ -1,9 +1,9 @@
 """Create the PDFjs template."""
 
-from pkg_resources import resource_filename
+import os
 
-SOURCE_FILE = resource_filename("via", "static/vendor/pdfjs-2/web/viewer.html")
-TARGET_FILE = resource_filename("via", "templates/pdf_wrapper.html.jinja2")
+SOURCE_FILE = os.path.abspath("via/static/vendor/pdfjs-2/web/viewer.html")
+TARGET_FILE = os.path.abspath("via/templates/pdf_wrapper.html.jinja2")
```

Comment:
This is kind of a bad sign. Using `pkg_resources` is the right way to get at things inside a package in a `pwd` agnostic way. Not being able to call any code from the package in question in the product suggests our paths just aren't setup correctly.

A different way to fix this is to add `PYTHONPATH = .` to the `setenv` section of `tox.ini`

Reply:
Done